config.json

```diff
@@ -1402,6 +1402,19 @@
           "unsafe"
         ]
       },
+      {
+        "slug": "resistor-color",
+        "name": "Resistor Color",
+        "uuid": "51c31e6a-b7ec-469d-8a28-dd821fd857d2",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
```

Comment:
I'm curious: why did you assign this as difficulty 4 when the equivalent C exercise has it as difficulty 1?

Reply:
I think it's *a bit* more difficult because of the extra work expected. Are there difficulty levels between 1 and 4?

Reply:
Changed it to 1 now.